app/assets/stylesheets/orders.css.scss

```diff
@@ -22,14 +22,94 @@
   }
 }
 
-.payment-form {
-  width: 30%;
+.checkout {
+  @include span-columns(6);
 
-  .alert {
-    width: 950px;
+  #{$all-text-inputs} {
+    background: transparent;
+    border: none;
+    box-shadow: none;
+    height: auto;
+    line-height: 1.5em;
+    margin: 0;
+    padding: 1.25em 0;
+    width: 65%;
   }
 
-  fieldset {
-    background: none;
+  h5 {
+    text-transform: uppercase;
+  }
+
+  label {
+    color: #555;
+    cursor: pointer;
+    float: left;
+    font-weight: 500;
+    line-height: 1.5em;
+    margin-bottom: 0;
+    padding-bottom: 1.25em;
+    padding-top: 1.25em;
+    width: 35%;
+  }
+}
+
+.fieldset {
+  @include border-color(#d8d8d8 #d8d8d8 #d8d8d8 #d8d8d8);
+  @include border-top-radius(4px);
+  @include border-bottom-radius(4px);
+  @include border-style(solid solid solid solid);
+  @include border-width(1px 1px 1px 1px);
+  @include margin(null null 1.5em null);
+}
+
+.field {
+  @include border-color(#dfdfdf null null null);
+  @include border-style(solid null null null);
+  @include border-width(1px null null null);
+  @include padding(null 1.25em null 1.25em);
+
+  display: block;
+  position: relative;
+
+  &:after {
+    clear: both;
+  }
+
+  &:after,
+  &:before {
+    content: "";
+    display: table;
+  }
+
+  &:first-child {
+    border-top: none;
+  }
+}
+
+.select-group {
+  float: left;
+  position: relative;
+  width: 65%;
+
+  #{$all-text-inputs} {
+    float: right;
+    width: 44%;
+
+    &:first-child {
+      float: left;
+    }
+  }
+
+  &:after {
+    content: "";
+    display: block;
+    background: #e5e5e5;
```

Comment:
Color literals like `#e5e5e5` should only be used in variable declarations; they should be referred to via variable everywhere else.